roles/edpm_tripleo_cleanup/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+edpm_old_tripleo_services: []
+
+# Remove unit files
+remove_files: false
```

Comment:
We probably want to prefix this with edpm_ and make a bit more specific to the role. E.g. edpm_remove_tripleo_service_files .

Also I would make the default true as I believe we should not keep unusable clutter around after the cleanup role.

Reply:
I'd rather not make it a default, for exactly the reasons I've stated. Those files may be managed by their packages, the removal itself is, unlike masking, disabling etc. irreversible, and the space saved is on order of KBs.

At very best we are freeing equivalent of a small text document, at worst we are inconveniencing administrator.

Reply:
I think @karelyatin showed above that these are not part of a package. 

It is not about saving space it is about to remove confusion. At the point of time when the cleanup role runs the node is not managed by tripleo and will never be able to go back to tripleo as we ran the 18 deployment on it. So having tripleo specific files on the node might wrongly suggest that this node is managed by tripleo or can managed by tripleo in the future.

Reply:
Very well, it's now the default behavior.